src/status_im/contexts/communities/overview/events_test.cljs

```diff
@@ -42,10 +42,11 @@
         signatures          ["11111" "222222"]
         expected            {:fx [[:json-rpc/call
                                    [{:method      "wakuext_requestToJoinCommunity"
-                                     :params      [{:communityId       community-id
-                                                    :signatures        signatures
-                                                    :addressesToReveal addresses-to-reveal
-                                                    :airdropAddress    "0x1"}]
+                                     :params      [{:communityId          community-id
+                                                    :signatures           signatures
+                                                    :addressesToReveal    addresses-to-reveal
+                                                    :shareFutureAddresses share-future-addresses
```

Comment:
`share-future-addresses` is also undefined here.

Reply:
Fixed.